# waypoints list
waypoints_list:
    - id: 0
      x: 0.00
      y: 0.00

    - id: 1
      x: 1.50
      y: 0.00

    - id: 2
      x: 2.50
      y: 0.00

    - id: 3
      x: 3.50
      y: 0.00

    - id: 4
      x: 4.50
      y: 0.00

    - id: 5
      x: 5.50
      y: 0.00

    - id: 6
      x: 6.50
      y: 0.00

    - id: 7
      x: 7.50
      y: 0.00